app.py

```diff
@@ -136,6 +145,76 @@ def prepare_response_received(random_delay, random_case):
     messages_to_send.append(message)
 
 
+def prepare_response_received_pqrs(random_delay, random_case):
+    tx_id = str(uuid.uuid4())
+
+    message_contents = {
+        "channel": "PQRS",
+        "dateTime": f"{datetime.utcnow().isoformat()}Z",
+        "questionnaireId": random_case['qid'],
+        "transactionId": str(uuid.uuid4()),
+        "type": "RESPONSE_RECEIVED"
+    }
+
+    message = {
+        "type": "PUBSUB_EQ",
+        "topic": Config.OFFLINE_RECEIPT_TOPIC_NAME,
+        "project": Config.OFFLINE_RECEIPT_TOPIC_PROJECT_ID,
+        "delay": random_delay,
+        "message_body": json.dumps(message_contents),
+        "tx_id": tx_id
+    }
+
+    messages_to_send.append(message)
+
+
+def prepare_response_received_qm(random_delay, random_case):
```

Comment:
this is identical to `prepare_response_received_pqrs` except for the channel - would be good to have a single function which takes the channel as a parameter